test/e2e/specs/onboarding/setup__start-writing.ts

```diff
@@ -0,0 +1,154 @@
+/**
+ * @group calypso-release
+ */
+import {
+	DataHelper,
+	UserSignupPage,
+	BrowserManager,
+	NewUserResponse,
+	RestAPIClient,
+	DomainSearchComponent,
+	EditorPage,
+	CartCheckoutPage,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+import { apiCloseAccount } from '../shared';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Start Writing Tailored Onboarding' ), () => {
```

Comment:
**issue:non-blocking** - while not documented, we are deprecating the use of the `DataHelper.createSuiteTitle()` helper method.